scripts/prestart_hook.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Define the URL and parameters
+PLATFORM="docker"
+URL="https://hyperswitch.gateway.scarf.sh/$PLATFORM"
+VERSION="unknown"
+STATUS="initiated"
+
+# Send the GET request
+curl -G "$URL" --data-urlencode "version=$VERSION" --data-urlencode "status=$STATUS"
```

Comment:
Prefer to use the longer version of flags in scripts and documentation.

```suggestion
curl --get "$URL" --data-urlencode "version=$VERSION" --data-urlencode "status=$STATUS"
```